test/client/fixtures/sandbox/code-instrumentation/location-test.js

```diff
@@ -157,6 +178,54 @@ test('special pages (GH-339)', function () {
     destLocation.forceLocation(storedForcedLocation);
 });
 
+test('should add the trailing slash to location.href if url consist of origin (GH-1426)', function () {
+    var storedForcedLocation = destLocation.getLocation();
+
+    function checkTrailingSlash (urls, trailingSlashIsNeeded) {
+        urls.forEach(function (url) {
+            destLocation.forceLocation(urlUtils.getProxyUrl(url));
+
+            var windowMock = {
+                location: {},
+                document: document
+            };
+
+            var locationWrapper = new LocationWrapper(windowMock);
+
+            strictEqual(locationWrapper.href, url + (trailingSlashIsNeeded ? '/' : ''));
+        });
+    }
+
+    checkTrailingSlash(SHOULD_ADD_TRAILING_SLASH, true);
+    checkTrailingSlash(SHOULD_NOT_ADD_TRAILING_SLASH, false);
+
+    destLocation.forceLocation(storedForcedLocation);
+});
+
+test('should add the trailing slash to location when navigateTo() is called (GH-1426)', function () {
+    var getWindowMock = function () {
+        return {
+            navigateToUrl: navigateTo,
+
+            win: {
+                location: ''
+            },
+        };
+    };
+
+    var windowMock = getWindowMock();
```

Comment:
Bad test algorithm.
`window` does not contain the `navigateTo` method.
Rewrite as:
```js
var storedWindow = hammerhead.win;

hammerhead.win = windowMock;

test action with hammerhead.navigateTo()

restore hammerhead.win = storedWindow;
```

Reply:
Done.